l10n-pocket/en/pocket/tos.ftl

```diff
@@ -0,0 +1,128 @@
+
+pocket-tos-the-pocket-technologies-including-without = The { -brand-name-pocket-technologies } (including, without limitation, our application, software, products, services and website) are provided "as is" and with all faults. We make no representation or warranty whatsoever regarding the completeness, accuracy, adequacy, suitability, functionality, availability, or operation of the { -brand-name-pocket-technologies }. you acknowledge that we do not have control over your use of the { -brand-name-pocket-technologies }, and we do not warrant the performance or results that may be obtained through your use of the { -brand-name-pocket-technologies }. you assume all risks and responsibility for your use of the { -brand-name-pocket-technologies } and for any loss of or errors in any data or information. to the full extent permissible by applicable law, we disclaim all warranties, express or implied, including, but not limited to, the implied warranties of merchantability, fitness for a particular purpose, and noninfringement, and any warranties arising through course of dealing or usage of trade. without limiting the foregoing, we neither warrant nor represent that your use of the { -brand-name-pocket-technologies } will not infringe the rights of any third parties, nor that the { -brand-name-pocket-technologies } will be available for your access or use, nor that operation of the { -brand-name-pocket-technologies } will be error free or uninterrupted. please note that some jurisdictions may not allow the exclusion of implied warranties, so some of the above exclusions may not apply to you. in addition, you may also have other rights which vary from state to state.
+
+pocket-tos-there-are-inherent = There are inherent risks in relying upon, using, or retrieving any information found on the internet, and we urge you to make sure you understand these risks before using the { -brand-name-pocket-technologies } or any website that provides you with access to the { -brand-name-pocket-technologies }. your use of the { -brand-name-pocket-technologies } is at your own risk. neither { -brand-name-mozilla } nor any of its parents, subsidiaries or affiliates, nor any of their employees, officers or directors, shall be liable for any direct, indirect, punitive, incidental, special, consequential, or other damages (including, without limitation, loss of data or information of any kind, loss of business, lost profits, interruption of business, cost of cover or any other damages) arising out of or in any way related to this agreement or the use or inability to use the { -brand-name-pocket-technologies }, whether based on contract, tort, strict liability or otherwise, even if we have been advised of the possibility of such damages. please note that some jurisdictions may not allow the exclusion of certain damages, so some of the above exclusions may not apply to you.
```

Comment:
**Nit:** Capitalize after period:
```suggestion
pocket-tos-there-are-inherent = There are inherent risks in relying upon, using, or retrieving any information found on the internet, and we urge you to make sure you understand these risks before using the { -brand-name-pocket-technologies } or any website that provides you with access to the { -brand-name-pocket-technologies }. Your use of the { -brand-name-pocket-technologies } is at your own risk. Neither { -brand-name-mozilla } nor any of its parents, subsidiaries or affiliates, nor any of their employees, officers or directors, shall be liable for any direct, indirect, punitive, incidental, special, consequential, or other damages (including, without limitation, loss of data or information of any kind, loss of business, lost profits, interruption of business, cost of cover or any other damages) arising out of or in any way related to this agreement or the use or inability to use the { -brand-name-pocket-technologies }, whether based on contract, tort, strict liability or otherwise, even if we have been advised of the possibility of such damages. Please note that some jurisdictions may not allow the exclusion of certain damages, so some of the above exclusions may not apply to you.
```